README.md

```diff
@@ -1 +1,30 @@
-# littil-frontend
\ No newline at end of file
+# LITTIL website
+
+This project contains the sourcecode of the LITTIL website.
+
+## Setup your local environment
+For more information about setting up your local development environment, please read [set up your development environment](https://github.com/Devoxx4Kids-NPO/littil-frontend/blob/main/docs/set-up-your-development-environment.md).
```

Comment:
link doesn't seem to work.

Reply:
I've added a link referring to the main branch, which doesn't exist yet. If there is a way to refer to an md file in the repository without using absolute URL's then please let me know so I can fix this.

LTDR; the URL should work when this PR is merged to the main branch.

Reply:
Apparently you can use relative paths on Github. Fixed the URI. It works now.